client/src/components/pages/LandingPage.js

```diff
@@ -7,13 +10,61 @@ import OtherNavBar from '../common/OtherNavBar';
 import { LazyLoadImage } from 'react-lazy-load-image-component';
 import { withStyles } from '@material-ui/core/styles';
 import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
 
-const style = {
-  position: 'absolute',
-  width: '100%',
-  margin: 0,
-  padding: 0
-};
+const style = (theme) => ({
+  container: {
+    height: `calc(100vh - ${64}px)`,
+    width: '100vw',
+    overflow: 'hidden',
+    [theme.breakpoints.between('xs', 'sm')]: {
+      width: '100vw'
+    },
+
+  },
+  subcontainer: {
+    position: 'absolute',
+    width: '100vw',
+    margin: 0,
+    padding: 0
+  },
+  text: {
+    position: 'absolute',
+    zIndex: 100,
+    top: 140,
+    left: '12%',
+  },
+  textH1: {
+    fontSize: '4rem',
+    fontWeight: 'bold',
+    fontFamily: 'Arial, Helvetica, sans-serif',
+    [theme.breakpoints.between('xs', 'sm')]: {
+      fontSize: '3.2rem'
+    },
+  },
+  textP: {
+    fontSize: '1.6rem',
+    width: '35%',
+    paddingLeft: 40,
+    paddingTop: 10,
+    fontFamily: 'Arial, Helvetica, sans-serif',
+    lineHeight: 1.5,
+    marginBlockStart: '1em',
+    marginBlockEnd: '1em',
+    [theme.breakpoints.between('xs', 'sm')]: {
+      width: '50%',
+      fontSize: '1.4rem'
+    },
+  },
+  backgroundImg: {
+    zIndex: -1,
+    width: '100vw',
+    height: '100vh',
+    marginTop: 0,
+
+  }
+
+});
```

Comment:
Thanks for converting the CSS to styled components! I was planning on refactoring this later on